whois-query/src/main/java/net/ripe/db/whois/query/acl/AccessControlListManager.java

```diff
@@ -148,6 +160,11 @@ private String getUserName(final String ssoToken, final OAuthSession oAuthSessio
         return null;
     }
 
+    @Nullable
+    private String getSsoTokenFromOauth(final OAuthSession oAuthSession){
```

Comment:
Why qury happens it always accounts for username why you need oauthsession here ? We should not add that differentiation while counting for personal objects